### ISSUE TITLE
Do not display wallet features that are unsupported on a given network.

### ISSUE THREAD

**Issue by 0xDaedalus** [tallycash/extension#2479]
When on the RSK Network - we will want to hide the swaps page and the NFT tab.

We probably want to do this with a whitelist in preparation of adding more networks that do not necessarily support swaps or NFTs.

Something like

```
export const NETWORKS_SUPPORTING_SWAPS = [ ... ]
export const NETWORKS_SUPPORTING_NFTS = [ ... ]
```

We'll also need to make sure that if a user is on the swaps / nft page and switches over to a network that does not support that feature - the app behaves gracefully.

**Comment by ahsan-javaiid** [comment]
Implementation done in RSK Integration PR: https://github.com/tallycash/extension/pull/2416

cc: @0xDaedalus, @alepc253

**Comment by 0xDaedalus** [comment]
Closed by #2416
